atlasdb-coordination-impl/src/main/java/com/palantir/atlasdb/coordination/keyvalue/KeyValueServiceCoordinationStore.java

```diff
@@ -0,0 +1,181 @@
+import com.palantir.atlasdb.coordination.SequenceAndBound;
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.CheckAndSetException;
+import com.palantir.atlasdb.keyvalue.api.CheckAndSetRequest;
+import com.palantir.atlasdb.keyvalue.api.KeyValueService;
+import com.palantir.atlasdb.keyvalue.api.Value;
+import com.palantir.atlasdb.keyvalue.impl.CheckAndSetResult;
+import com.palantir.atlasdb.keyvalue.impl.ImmutableCheckAndSetResult;
+import com.palantir.atlasdb.protos.generated.TableMetadataPersistence;
+import com.palantir.atlasdb.table.description.ColumnMetadataDescription;
+import com.palantir.atlasdb.table.description.ColumnValueDescription;
+import com.palantir.atlasdb.table.description.DynamicColumnDescription;
+import com.palantir.atlasdb.table.description.NameComponentDescription;
+import com.palantir.atlasdb.table.description.NameMetadataDescription;
+import com.palantir.atlasdb.table.description.TableMetadata;
+import com.palantir.atlasdb.table.description.ValueType;
+import com.palantir.atlasdb.transaction.api.ConflictHandler;
+import com.palantir.logsafe.SafeArg;
+import com.palantir.remoting3.ext.jackson.ObjectMappers;
+
+public class KeyValueServiceCoordinationStore implements CoordinationStore {
+    private static final Logger log = LoggerFactory.getLogger(KeyValueServiceCoordinationStore.class);
+
+    private static final TableMetadata COORDINATION_TABLE_METADATA = new TableMetadata(
+            NameMetadataDescription.create(ImmutableList.of(
+                    new NameComponentDescription.Builder()
+                            .componentName("sequence")
+                            .type(ValueType.BLOB)
+                            .logSafety(TableMetadataPersistence.LogSafety.SAFE)
+                            .build())),
+            new ColumnMetadataDescription(new DynamicColumnDescription(
+                    NameMetadataDescription.create(
+                            ImmutableList.of(
+                                    new NameComponentDescription.Builder()
+                                            .componentName("sequenceNumber")
+                                            .logSafety(TableMetadataPersistence.LogSafety.SAFE)
+                                            .type(ValueType.VAR_LONG)
+                                            .build())),
+                    ColumnValueDescription.forType(ValueType.VAR_LONG))),
+            ConflictHandler.IGNORE_ALL,
+            TableMetadataPersistence.CachePriority.WARM,
+            false,
+            0,
+            false,
+            TableMetadataPersistence.SweepStrategy.NOTHING, // we do our own cleanup
+            false,
+            TableMetadataPersistence.LogSafety.SAFE);
+    private static final long WRITE_TIMESTAMP = 0L;
+    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.newServerObjectMapper();
+
+    private final KeyValueService kvs;
+    private final byte[] coordinationSequence;
```

Comment:
It is the row key used in the table - for transactions2 we only plan on using one sequence, but if we want more sequences in the future we don't want to use additional tables since that has some other overheads.

(This will be in the TODO item I mention above on javadoc of the schema)